container-images/candlepin/broker.xml.tpl

```diff
@@ -10,7 +10,7 @@
 
         <acceptors>
             <acceptor name="in-vm">vm://0</acceptor>
-            <acceptor name="stomp">tcp://localhost:61613?protocols=STOMP;useEpoll=false;sslEnabled=true;trustStorePath=/etc/candlepin/certs/truststore;trustStorePassword={{ candlepin_keystore_password }};keyStorePath=/etc/candlepin/certs/keystore;keyStorePassword={{ candlepin_keystore_password }};needClientAuth=true</acceptor>
+            <acceptor name="stomp">tcp://localhost:61613?protocols=STOMP;useEpoll=false;sslEnabled=true;trustStorePath=/etc/candlepin/certs/truststore;trustStorePassword=${CANDLEPIN_KEYSTORE_PASSWORD};keyStorePath=/etc/candlepin/certs/keystore;keyStorePassword=${CANDLEPIN_KEYSTORE_PASSWORD};needClientAuth=true</acceptor>
```

Comment:
I think https://activemq.apache.org/components/artemis/documentation/latest/configuring-transports.html#configuring-netty-ssl is the relevant documentation and I don't see a `keystorePasswordFile` option (or similar).

It does mention you can provide the `javax.net.ssl.keyStorePassword` system property as well. I wonder if Candlepin can reuse that same value. Perhaps we can end up with providing one big properties file with all secrets. Even if we don't directly expose that but still template it, it could be interesting to separate it out.